CMakeModules/AFconfigure_forge_submodule.cmake

```diff
@@ -34,6 +33,8 @@ if(AF_BUILD_FORGE)
       $<TARGET_FILE:forge>
       $<$<PLATFORM_ID:Linux>:$<TARGET_SONAME_FILE:forge>>
       $<$<PLATFORM_ID:Darwin>:$<TARGET_SONAME_FILE:forge>>
+      $<$<PLATFORM_ID:Linux>:$<TARGET_LINKER_FILE:forge>>
```

Comment:
This is needed since runtime dependency module is not taking into account the soname extensions yet.